### PR TITLE
Fix numeric locale configuration.

### PR DESCRIPTION
**Pull request by SteveIreland**
This is another attempt to fix the locale configuration issue #2574.

### CODE REVIEW

File: application/controllers/Config.php [hunk 1]
```diff
@@ -308,10 +309,21 @@ public function save_general()
 
 	public function ajax_check_number_locale()
 	{
-		$number_locale = $this->input->post('number_locale');
-		$fmt = new \NumberFormatter($number_locale, \NumberFormatter::CURRENCY);
-		$currency_symbol = empty($this->input->post('currency_symbol')) ? $fmt->getSymbol(\NumberFormatter::CURRENCY_SYMBOL) : $this->input->post('currency_symbol');
-		$currency_code = empty($this->input->post('currency_code')) ? $fmt->getTextAttribute(\NumberFormatter::CURRENCY_CODE) : $this->input->post('currency_code');
+        $number_locale = $this->input->post('number_locale');
```
Comment (daN4cat) [inline application/controllers/Config.php]: There is an indentation issue with the newly added code.

**Comment by odiea** [comment]
I can actually see changes when selecting a different language code  for localization now. It also appears to be saved with the Submit button and the currency symbol can be changed after selecting the new localization. Thanks @SteveIreland

File: application/controllers/Config.php (hunk 1 above)
Comment (SteveIreland) [reply application/controllers/Config.php]: Fixed

**Comment by SteveIreland** [comment]
Thanks for testing it @odiea!

**Comment by daN4cat** [comment]
Good job, @SteveIreland

**Pull request merged**